Runtime/Properties/ISnapZoneProperty.cs

```diff
@@ -24,7 +23,7 @@ public interface ISnapZoneProperty : ISceneObjectProperty, ILockable
         /// <summary>
         /// Snap zone object.
         /// </summary>
-        GameObject SnapZoneObject { get; }
+        ISnapZone SnapZoneObject { get; }
```

Comment:
API break!? 😱🤔

Reply:
Yes, we talked about it, as far as I thought. Is this actually a problem? Because the workaround would be still returning the GameObject and extracting the ISnapZone then, which is BS.

Reply:
I know the current implementation sucks but we don't break APIs unless we release a major version.
I would rather obsolete `SnapZoneObject` and introduce a direct `ISnapZone` property.

Reply:
It's an interface you CANT change it in any meaning without breaking API